### ISSUE TITLE
Documentation on setting up a multi-cloud cluster?

### ISSUE THREAD

**Issue by zplizzi** [squat/kilo#219]
The docs clearly imply that one of the main uses of Kilo is setting up a multi-cloud cluster ("Kilo enables the operation of multi-cloud clusters"), but I can't anywhere find documentation of how to actually do that.

As an example, I have set up a cluster in EC2 using `kubeadm` with a single control plane node and a couple workers. Then I've started a GCP instance that I want to connect to the cluster using Kilo. How do I do this?? I can't find anywhere in the documentation that explains this step.

I know I can just open up my control plane node's IP and ports to the internet and let the GCP node connect directly with `kubeadm join PUBLIC_IP:PORT` - is that what I'm supposed to do? I'm not sure if that's safe to do. But in the kilo docs it says "The nodes in the mesh will require an open UDP port in order to communicate" - which I took to mean that there's a way to do this by only opening a single port that Kilo will use to make the connection for me somehow.

**Comment by zplizzi** [comment]
I know #165 is tracking requests for better documentation - just hoping maybe you can quickly point me in the right direction here :)

**Comment by zplizzi** [comment]
Ah, I just found #11, I think that has the explanation I was looking for.

**Comment by leonnicolas** [comment]
Added this to #165

**Comment by zplizzi** [comment]
Didn't quite get this working, but the instructions in #11 definitely helped. It seems like kubeadm doesn't play super well with AWS EC2 nodes having different public vs private IPs.. had to do some [hacky stuff](https://github.com/kubernetes/kubeadm/issues/203#issuecomment-478206793) to get that to work (to join a node from a different region that wasn't networked to the main cluster with VPC peering). Also couldn't get it to work with just kilo as the networking plugin - not sure what I was doing wrong there. Got farther when I used flannel and then put kilo on top of that. Unfortunately I had to stop working on this and shut down my test nodes so this will have to wait for another day. Looking forward to checking out the new documentation when it goes up :)
